From b616d5b4f2b132cf8226e96858b1ef37dc9127cf Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Mon, 6 May 2019 21:22:30 -0600
Subject: [PATCH] [SQUASH][DNM] Merge tag 'android-9.0.0_r37' into
 staging/lineage-16.0_merge-android-9.0.0_r37

Android 9.0.0 Release 37 (PQ3A.190505.002)

* tag 'android-9.0.0_r37':
  Import translations. DO NOT MERGE
  Import translations. DO NOT MERGE
  Fix test to use better flag.
  Fix bug in warning activity intent.

https://github.com/haggertk/android_packages_apps_StorageManager/tree/lineage-16.0_merge-android-9.0.0_r37

Change-Id: I793a226db46cb4a47ca6e40fdcb5ddadcb4c1001
---
 res/values-en-rXC/arrays.xml                  |  6 +-
 res/values-en-rXC/strings.xml                 | 76 +++++++++----------
 res/values-es/strings.xml                     |  2 +-
 res/values-nl/strings.xml                     |  2 +-
 ...omaticStorageManagementJobServiceTest.java |  4 +-
 .../automatic/NotificationController.java     |  4 +-
 6 files changed, 48 insertions(+), 46 deletions(-)

diff --git a/res/values-en-rXC/arrays.xml b/res/values-en-rXC/arrays.xml
index c651ee5..733d5ed 100644
--- a/res/values-en-rXC/arrays.xml
+++ b/res/values-en-rXC/arrays.xml
@@ -17,8 +17,8 @@
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
   <string-array name="automatic_storage_management_days">
-    <item msgid="4262697175462957945">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‏‏‎‏‏‎‎‏‎‏‎‎‎‎‎‏‎‎‏‎‎‎‏‏‏‏‏‏‏‎‎‎‏‎‎‎‎‏‎‎‎‎‏‏‏‏‏‎‏‎‎‏‏‎‏‏‏‏‎‎‏‎Over 30 days old‎‏‎‎‏‎"</item>
-    <item msgid="2920779420892632946">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‎‏‎‎‎‏‎‎‎‏‎‎‎‏‎‏‏‎‎‎‏‎‏‏‎‏‎‎‎‏‎‎‎‏‎‎‏‏‎‎‏‎‎‏‏‏‎‎‏‎‏‏‏‎‏‏‏‎‎‏‎‎Over 60 days old‎‏‎‎‏‎"</item>
-    <item msgid="1402081670011072896">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‏‏‎‎‏‏‎‏‏‏‎‏‎‏‎‎‏‏‎‎‎‏‏‏‎‏‏‎‎‏‏‎‏‎‏‏‏‏‎‎‎‏‏‎‎‎‏‎‏‏‎‎‎‏‏‎‎‎‎‎‎‎‎Over 90 days old‎‏‎‎‏‎"</item>
+    <item msgid="4262697175462957945">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‏‏‎‏‏‎‎‏‎‏‎‎‎‎‎‏‎‎‏‎‎‎‏‏‏‏‏‏‏‎‎‎‏‎‎‎‎‏‎‎‎‎‏‏‏‏‏‎‏‎‎‏‏‎‏‏‏‏‎‎‏‎Over 30 days old‎‏‎‎‏‎"</item>
+    <item msgid="2920779420892632946">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‎‏‎‎‎‏‎‎‎‏‎‎‎‏‎‏‏‎‎‎‏‎‏‏‎‏‎‎‎‏‎‎‎‏‎‎‏‏‎‎‏‎‎‏‏‏‎‎‏‎‏‏‏‎‏‏‏‎‎‏‎‎Over 60 days old‎‏‎‎‏‎"</item>
+    <item msgid="1402081670011072896">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‏‏‎‎‏‏‎‏‏‏‎‏‎‏‎‎‏‏‎‎‎‏‏‏‎‏‏‎‎‏‏‎‏‎‏‏‏‏‎‎‎‏‏‎‎‎‏‎‏‏‎‎‎‏‏‎‎‎‎‎‎‎‎Over 90 days old‎‏‎‎‏‎"</item>
   </string-array>
 </resources>
diff --git a/res/values-en-rXC/strings.xml b/res/values-en-rXC/strings.xml
index f5982f1..087bab8 100644
--- a/res/values-en-rXC/strings.xml
+++ b/res/values-en-rXC/strings.xml
@@ -16,52 +16,52 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="app_name" msgid="963247385700831716">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‎‏‏‎‏‎‏‎‏‏‏‏‎‎‎‏‎‎‏‎‎‎‏‏‎‎‏‎‎‏‏‎‎‏‏‏‏‏‎‎‎‎‏‎‏‏‎‎‏‎‎‎‏‏‏‏‎‎‏‎‎‎Storage Manager‎‏‎‎‏‎"</string>
-    <string name="cancel" msgid="7021218262867558825">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‎‎‎‎‎‏‏‎‎‏‏‏‏‎‏‏‎‎‎‎‎‏‎‏‏‎‏‎‎‏‏‎‎‎‎‎‏‏‎‏‎‏‎‎‏‎Cancel‎‏‎‎‏‎"</string>
-    <string name="storage_menu_free" msgid="1878247401436882778">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‏‏‏‎‏‎‎‎‎‏‎‎‎‎‏‏‏‎‎‎‎‎‎‎‎‎‏‏‏‏‎‏‏‎‏‎‏‎‏‏‏‏‎‎‏‎‎‎‏‎‎‏‏‏‎‏‎‏‏‎‏‎‎Free up space‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_title" msgid="3526170325226275927">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‏‎‎‎‎‏‏‏‎‏‏‏‏‎‏‏‏‏‎‎‏‎‎‏‏‏‎‏‎‎‏‏‏‎‏‎‎‏‎‏‎‎‎‏‎‏‏‎‎‎‏‎‎‎‏‎‏‎‏‏‏‎Remove items‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_app_summary" msgid="4904590040180275237">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‎‎‎‏‎‎‎‎‎‏‎‎‎‎‏‎‎‏‏‎‏‎‏‎‏‎‎‏‏‏‏‏‏‎‏‎‎‏‏‏‎‏‏‏‎‏‎‎‏‏‏‎‎‎‎‎‏‎‎‏‎‏‎‎‏‎‎‏‏‎<xliff:g id="DAYS">%1$d</xliff:g>‎‏‎‎‏‏‏‎ days ago‎‏‎‎‏‎"</string>
+    <string name="app_name" msgid="963247385700831716">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‎‏‏‎‏‎‏‎‏‏‏‏‎‎‎‏‎‎‏‎‎‎‏‏‎‎‏‎‎‏‏‎‎‏‏‏‏‏‎‎‎‎‏‎‏‏‎‎‏‎‎‎‏‏‏‏‎‎‏‎‎‎Storage Manager‎‏‎‎‏‎"</string>
+    <string name="cancel" msgid="7021218262867558825">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‎‎‎‎‎‏‏‎‎‏‏‏‏‎‏‏‎‎‎‎‎‏‎‏‏‎‏‎‎‏‏‎‎‎‎‎‏‏‎‏‎‏‎‎‏‎Cancel‎‏‎‎‏‎"</string>
+    <string name="storage_menu_free" msgid="1878247401436882778">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‏‏‏‎‏‎‎‎‎‏‎‎‎‎‏‏‏‎‎‎‎‎‎‎‎‎‏‏‏‏‎‏‏‎‏‎‏‎‏‏‏‏‎‎‏‎‎‎‏‎‎‏‏‏‎‏‎‏‏‎‏‎‎Free up space‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_title" msgid="3526170325226275927">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‏‎‎‎‎‏‏‏‎‏‏‏‏‎‏‏‏‏‎‎‏‎‎‏‏‏‎‏‎‎‏‏‏‎‏‎‎‏‎‏‎‎‎‏‎‏‏‎‎‎‏‎‎‎‏‎‏‎‏‏‏‎Remove items‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_app_summary" msgid="4904590040180275237">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‎‎‎‏‎‎‎‎‎‏‎‎‎‎‏‎‎‏‏‎‏‎‏‎‏‎‎‏‏‏‏‏‏‎‏‎‎‏‏‏‎‏‏‏‎‏‎‎‏‏‏‎‎‎‎‎‏‎‎‏‎‏‎‎‏‎‎‏‏‎<xliff:g id="DAYS">%1$d</xliff:g>‎‏‎‎‏‏‏‎ days ago‎‏‎‎‏‎"</string>
     <!-- no translation found for deletion_helper_app_summary_item_size (3770886184921427886) -->
     <skip />
-    <string name="deletion_helper_app_summary_never_used" msgid="2695210890063792461">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‎‎‏‎‏‎‏‏‎‎‏‏‏‎‏‎‏‎‎‎‎‎‎‎‎‎‏‏‎‎‏‎‏‎‎‎‏‎‎‎‎‎‏‎‎‎‏‎‎‏‎‎‏‎‏‎‎‏‏‎‏‎Not used in last year‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_app_summary_unknown_used" msgid="8576377054665785558">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‏‏‏‎‎‎‎‎‏‎‏‎‏‏‎‏‎‎‏‎‏‏‏‏‏‎‏‏‏‎‏‏‎‏‏‎‎‏‎‏‎‏‎‎‏‎‎‎‎‎‎‏‏‎‏‎‏‏‎‎Not sure when last used‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_free_button" msgid="1760529213407548661">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‏‏‏‎‎‎‎‏‏‎‏‏‏‎‏‎‏‎‎‏‏‏‏‏‏‏‏‏‎‏‏‏‏‎‏‏‎‏‏‏‎‏‏‎‏‎‏‏‎‏‎‏‎‎‏‏‏‏‎‏‎‏‎Free up ‎‏‎‎‏‏‎<xliff:g id="FREEABLE">%1$s</xliff:g>‎‏‎‎‏‏‏‎‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_photos_title" msgid="2602723121486729972">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‎‎‏‎‎‎‎‎‏‏‏‏‎‏‎‏‏‏‎‏‎‏‏‏‎‎‎‏‏‏‎‏‎‎‎‎‎‏‏‏‎‏‎‎‏‏‎‎‏‎‎‏‎‏‏‏‏‎‏‎‎‎Backed up photos &amp; videos‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_photos_age_summary" msgid="1820871709448371984">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‏‏‏‎‎‏‎‏‎‎‎‏‎‏‎‎‎‎‏‎‎‏‎‎‏‎‏‏‎‎‎‎‎‏‏‏‏‏‎‎‏‏‏‎‏‎‎‏‏‏‏‏‏‏‎‎‎‏‎‎‎‎‎Over 30 days old‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_app_summary_never_used" msgid="2695210890063792461">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‎‎‏‎‏‎‏‏‎‎‏‏‏‎‏‎‏‎‎‎‎‎‎‎‎‎‏‏‎‎‏‎‏‎‎‎‏‎‎‎‎‎‏‎‎‎‏‎‎‏‎‎‏‎‏‎‎‏‏‎‏‎Not used in last year‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_app_summary_unknown_used" msgid="8576377054665785558">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‏‏‏‎‎‎‎‎‏‎‏‎‏‏‎‏‎‎‏‎‏‏‏‏‏‎‏‏‏‎‏‏‎‏‏‎‎‏‎‏‎‏‎‎‏‎‎‎‎‎‎‏‏‎‏‎‏‏‎‎Not sure when last used‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_free_button" msgid="1760529213407548661">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‏‏‏‎‎‎‎‏‏‎‏‏‏‎‏‎‏‎‎‏‏‏‏‏‏‏‏‏‎‏‏‏‏‎‏‏‎‏‏‏‎‏‏‎‏‎‏‏‎‏‎‏‎‎‏‏‏‏‎‏‎‏‎Free up ‎‏‎‎‏‏‎<xliff:g id="FREEABLE">%1$s</xliff:g>‎‏‎‎‏‏‏‎‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_photos_title" msgid="2602723121486729972">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‎‎‏‎‎‎‎‎‏‏‏‏‎‏‎‏‏‏‎‏‎‏‏‏‎‎‎‏‏‏‎‏‎‎‎‎‎‏‏‏‎‏‎‎‏‏‎‎‏‎‎‏‎‏‏‏‏‎‏‎‎‎Backed up photos &amp; videos‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_photos_age_summary" msgid="1820871709448371984">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‏‏‏‎‎‏‎‏‎‎‎‏‎‏‎‎‎‎‏‎‎‏‎‎‏‎‏‏‎‎‎‎‎‏‏‏‏‏‎‎‏‏‏‎‏‎‎‏‏‏‏‏‏‏‎‎‎‏‎‎‎‎‎Over 30 days old‎‏‎‎‏‎"</string>
     <!-- no translation found for deletion_helper_photos_summary (3194825464552700889) -->
     <skip />
-    <string name="deletion_helper_downloads_title" msgid="8002058095019254694">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‏‏‏‏‎‎‎‎‏‏‎‏‎‎‎‎‎‏‎‏‎‏‏‏‎‎‎‎‏‏‎‏‎‎‎‏‏‎‏‎‏‎‎‏‎‏‏‎‎‎‏‏‏‎‏‎‎‏‏‎‎Downloads‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_downloads_title" msgid="8002058095019254694">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‏‏‏‏‎‎‎‎‏‏‎‏‎‎‎‎‎‏‎‏‎‏‏‏‎‎‎‎‏‏‎‏‎‎‎‏‏‎‏‎‏‎‎‏‎‏‏‎‎‎‏‏‏‎‏‎‎‏‏‎‎Downloads‎‏‎‎‏‎"</string>
     <!-- no translation found for deletion_helper_downloads_category_summary (8545989355108311347) -->
     <skip />
     <!-- no translation found for deletion_helper_downloads_summary_empty (3988054032360371887) -->
     <skip />
-    <string name="deletion_helper_clear_dialog_title" msgid="9169670752655850967">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‏‏‏‏‎‏‎‎‎‎‎‏‎‎‏‏‎‏‏‎‏‏‏‏‎‏‏‏‎‏‏‎‏‏‏‏‎‏‎‏‎‏‏‏‎‎‏‏‏‏‎‏‏‏‎‏‎‏‏‏‎Free up space‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_clear_dialog_message" msgid="5196490478187120700">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‎‎‏‎‎‎‎‎‎‏‏‏‎‏‏‎‏‎‎‏‎‎‏‎‎‎‏‏‏‎‏‏‎‎‎‏‏‏‎‎‎‎‏‎‏‏‎‎‎‎‏‎‎‎‎‎‏‏‏‏‎‎‎‎‏‎‎‏‏‎<xliff:g id="CLEARABLE_BYTES">%1$s</xliff:g>‎‏‎‎‏‏‏‎ of content will be removed from your device‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_clear_dialog_remove" msgid="863575755467985516">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‎‏‎‏‏‏‏‏‏‏‏‎‎‎‎‎‎‏‎‎‏‏‎‎‏‎‎‏‎‏‏‏‏‏‏‎‏‎‏‎‎‏‏‏‏‎‎‎‏‎‎‏‎‎‏‏‎‏‏‎‎‎Free up space‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_upsell_title" msgid="8512037674466762017">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‏‏‎‎‎‏‎‎‎‎‎‏‏‎‏‎‏‎‏‎‎‏‎‏‎‏‎‏‎‎‎‏‏‎‏‎‏‎‏‎‏‎‏‎‏‎‏‏‏‎‏‎‎‏‎‎‎‎‏‎Manage storage automatically?‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_upsell_summary" msgid="2301597713694474407">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‏‏‏‏‏‏‏‏‏‏‎‎‎‎‏‏‏‎‏‎‏‎‏‏‏‎‏‏‎‏‎‎‎‏‏‏‎‎‏‎‎‏‏‏‎‎‎‏‎‎‎‎‎‎‏‎‏‎‎‏‏‏‎‎‏‎‎‏‏‎<xliff:g id="USED">%1$s</xliff:g>‎‏‎‎‏‏‏‎ now free. Let Storage manager automatically free up space by removing backed up content from your device?‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_upsell_cancel" msgid="7084167642850053889">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‎‎‏‎‎‏‎‏‎‎‎‎‎‎‎‎‎‏‎‎‏‏‏‏‎‎‎‎‎‎‎‎‏‎‎‏‎‏‏‎‎‏‎‎‏‏‎‎‏‏‏‏‎‎‎‎‎‎‎‏‎No thanks‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_upsell_activate" msgid="7157360257760895976">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‎‎‏‏‎‏‎‏‎‏‎‎‎‎‎‎‏‏‎‏‎‎‏‏‏‎‏‏‏‎‏‎‏‏‎‏‎‏‎‏‎‏‏‎‎‎‏‎‎‎‏‏‏‏‏‎‏‎‎‎‎Turn on‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_apps_group_title" msgid="2578309121350341258">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‎‎‎‏‏‏‏‎‎‎‏‏‏‏‏‏‏‏‏‏‎‎‏‏‏‏‏‎‎‎‏‎‎‎‎‏‏‎‎‏‏‏‏‎‎‏‎‎‏‎‎‏‎‏‎‎‎‏‎‏‎‎Infrequently used apps‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_clear_dialog_title" msgid="9169670752655850967">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‏‏‏‏‎‏‎‎‎‎‎‏‎‎‏‏‎‏‏‎‏‏‏‏‎‏‏‏‎‏‏‎‏‏‏‏‎‏‎‏‎‏‏‏‎‎‏‏‏‏‎‏‏‏‎‏‎‏‏‏‎Free up space‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_clear_dialog_message" msgid="5196490478187120700">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‎‎‏‎‎‎‎‎‎‏‏‏‎‏‏‎‏‎‎‏‎‎‏‎‎‎‏‏‏‎‏‏‎‎‎‏‏‏‎‎‎‎‏‎‏‏‎‎‎‎‏‎‎‎‎‎‏‏‏‏‎‎‎‎‏‎‎‏‏‎<xliff:g id="CLEARABLE_BYTES">%1$s</xliff:g>‎‏‎‎‏‏‏‎ of content will be removed from your device‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_clear_dialog_remove" msgid="863575755467985516">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‎‏‎‏‏‏‏‏‏‏‏‎‎‎‎‎‎‏‎‎‏‏‎‎‏‎‎‏‎‏‏‏‏‏‏‎‏‎‏‎‎‏‏‏‏‎‎‎‏‎‎‏‎‎‏‏‎‏‏‎‎‎Free up space‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_upsell_title" msgid="8512037674466762017">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‏‏‎‎‎‏‎‎‎‎‎‏‏‎‏‎‏‎‏‎‎‏‎‏‎‏‎‏‎‎‎‏‏‎‏‎‏‎‏‎‏‎‏‎‏‎‏‏‏‎‏‎‎‏‎‎‎‎‏‎Manage storage automatically?‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_upsell_summary" msgid="2301597713694474407">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‏‏‏‏‏‏‏‏‏‏‎‎‎‎‏‏‏‎‏‎‏‎‏‏‏‎‏‏‎‏‎‎‎‏‏‏‎‎‏‎‎‏‏‏‎‎‎‏‎‎‎‎‎‎‏‎‏‎‎‏‏‏‎‎‏‎‎‏‏‎<xliff:g id="USED">%1$s</xliff:g>‎‏‎‎‏‏‏‎ now free. Let Storage manager automatically free up space by removing backed up content from your device?‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_upsell_cancel" msgid="7084167642850053889">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‎‎‏‎‎‏‎‏‎‎‎‎‎‎‎‎‎‏‎‎‏‏‏‏‎‎‎‎‎‎‎‎‏‎‎‏‎‏‏‎‎‏‎‎‏‏‎‎‏‏‏‏‎‎‎‎‎‎‎‏‎No thanks‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_upsell_activate" msgid="7157360257760895976">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‎‎‏‏‎‏‎‏‎‏‎‎‎‎‎‎‏‏‎‏‎‎‏‏‏‎‏‏‏‎‏‎‏‏‎‏‎‏‎‏‎‏‏‎‎‎‏‎‎‎‏‏‏‏‏‎‏‎‎‎‎Turn on‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_apps_group_title" msgid="2578309121350341258">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‎‎‎‏‏‏‏‎‎‎‏‏‏‏‏‏‏‏‏‏‎‎‏‏‏‏‏‎‎‎‏‎‎‎‎‏‏‎‎‏‏‏‏‎‎‏‎‎‏‎‎‏‎‏‎‎‎‏‎‏‎‎Infrequently used apps‎‏‎‎‏‎"</string>
     <!-- no translation found for deletion_helper_apps_group_summary (360315517252452356) -->
     <skip />
-    <string name="deletion_helper_automatic_title" msgid="4610119358490679299">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‏‏‏‏‏‏‏‏‏‏‎‏‎‎‏‏‎‏‏‏‏‎‎‏‎‎‎‎‏‏‎‎‎‏‏‏‎‏‏‏‏‏‏‎‏‎‎‎‏‏‎‎‎‎‎‎‎‎‎‏‏‎Automatic‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_manual_title" msgid="3905192395039688529">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‏‎‏‏‎‎‎‏‏‎‎‏‎‎‎‎‎‎‏‏‏‏‏‎‎‏‎‏‎‏‏‏‎‏‏‎‏‏‏‏‎‎‏‎‏‏‏‎‎‎‏‏‏‎‏‎‏‎‎‎‏‎Manual‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_preference_title" msgid="803484286872745892">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‎‏‎‏‏‎‎‏‎‎‏‏‎‏‎‎‎‏‏‎‎‏‎‏‏‎‎‏‏‏‎‏‎‏‏‎‏‏‎‏‎‏‎‎‎‎‏‏‏‎‏‏‏‏‎‏‎‎‏‎‎‎Free space now‎‏‎‎‏‎"</string>
-    <string name="automatic_storage_manager_service_label" msgid="648592272375278978">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‎‏‎‎‏‎‎‎‎‎‎‎‎‎‏‎‎‎‎‏‏‎‎‏‏‏‏‎‎‎‎‏‏‏‏‏‎‏‏‏‏‏‏‎‏‏‏‏‎‏‏‎‏‏‎‎‎‎‎‏‎‎Automatic Storage Management Service‎‏‎‎‏‎"</string>
-    <string name="automatic_storage_manager_notification_title" product="default" msgid="8480483265300964701">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‏‎‏‏‎‏‏‎‎‎‎‏‎‏‏‏‎‏‎‏‎‎‏‏‎‎‏‎‎‏‎‎‏‏‏‏‏‏‎‏‎‎‎‏‎‏‎‎‏‎‏‎‏‎‏‏‏‎‏‎Your phone is low on space‎‏‎‎‏‎"</string>
-    <string name="automatic_storage_manager_notification_title" product="tablet" msgid="3150907918531844467">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‎‏‎‏‏‏‎‏‏‏‎‏‎‎‏‎‎‎‏‏‎‎‎‎‎‏‏‏‎‏‏‎‎‎‏‎‎‎‎‎‎‏‏‏‎‎‏‎‏‏‎‎‏‎‏‏‏‎‎‏‏‎Your tablet is low on space‎‏‎‎‏‎"</string>
-    <string name="automatic_storage_manager_notification_summary" msgid="2267399886362245822">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‏‏‏‏‏‏‎‏‏‏‎‏‏‏‎‏‏‎‏‏‎‎‎‎‏‎‏‏‏‎‏‏‏‏‎‏‏‎‎‏‏‎‎‎‎‏‏‏‏‏‏‎‏‎‏‎‏‏‏‏‏‎‎Let Storage Manager free up space by automatically removing old photos and videos when your phone starts to get full again.‎‏‎‎‏‎"</string>
-    <string name="automatic_storage_manager_cancel_button" msgid="1826365037494962803">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‏‏‏‎‎‏‎‏‎‏‏‎‎‎‏‎‎‎‏‏‎‏‎‏‎‏‎‎‏‏‎‎‏‏‏‎‏‎‎‎‎‏‎‏‎‎‏‎‎‎‏‎‏‎‎‏‏‏‎‎‏‏‎No thanks‎‏‎‎‏‎"</string>
-    <string name="automatic_storage_manager_activate_button" msgid="6257070263061898041">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‎‏‎‏‏‎‏‏‎‏‎‏‎‏‏‎‎‏‎‏‎‎‎‏‎‎‎‎‎‎‏‎‏‏‏‏‎‎‎‏‏‏‎‏‏‎‏‏‏‎‎‎‏‏‎‎‏‏‏‎‎‏‎Turn on‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_photos_loading_title" msgid="2768067991066779772">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‎‎‏‏‎‎‏‏‎‏‎‏‎‎‎‏‎‎‏‏‏‎‎‏‎‏‏‎‎‎‎‎‎‏‎‎‏‏‎‏‎‏‎‏‎‎‎‏‎‎‎‎‎‎‏‏‏‏‏‎‎‎Backed up photos &amp; videos‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_photos_loading_summary" msgid="8203033249458245854">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‎‎‏‏‏‎‏‎‏‏‏‎‎‎‎‎‏‏‏‎‏‎‎‎‏‏‏‎‏‏‎‎‎‏‎‏‎‏‎‏‏‎‏‎‏‎‏‎‎‎‎‏‏‎‏‏‏‏‎‎Finding items…‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_no_threshold" msgid="6943179204098250444">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‎‎‎‎‎‏‎‏‏‎‏‏‎‎‏‎‎‎‎‎‏‎‏‏‎‎‏‏‎‎‎‏‏‎‎‎‎‎‏‎‎‏‏‏‏‏‎‏‎‏‏‎‏‏‎‎‏‏‎‎‎Show all items‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_default_threshold" msgid="8410389370069021113">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‏‎‎‏‎‏‏‎‏‏‏‏‎‏‏‎‏‎‎‏‎‎‏‎‎‏‎‎‏‏‎‏‎‎‏‏‎‏‎‏‏‏‏‎‏‏‏‎‎‎‏‏‎‏‏‏‎‎‏‎Hide recent items‎‏‎‎‏‎"</string>
-    <string name="deletion_helper_clear_dialog_message_first_time" msgid="686530413183529901">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‎‏‎‎‏‏‎‎‎‎‏‏‏‎‎‎‎‏‎‏‏‏‏‎‎‎‏‎‏‏‏‏‏‏‏‎‏‎‏‎‎‏‏‏‎‎‏‏‏‎‎‏‏‏‎‏‎‏‏‎‏‎‎‏‎‎‏‏‎<xliff:g id="CLEARABLE_BYTES">%1$s</xliff:g>‎‏‎‎‏‏‏‎ of content will be removed from your device‎‏‎‎‏‎"</string>
-    <string name="automatic_storage_manager_activation_warning" msgid="7657017408180001078">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‏‎‏‎‎‏‎‎‎‎‏‏‎‎‏‏‎‎‎‎‏‏‎‎‎‎‏‏‏‎‎‏‏‏‎‏‏‎‎‎‏‏‎‏‎‎‎‏‏‎‎‏‎‎‏‏‎‏‏‎‎Your storage is now being managed by the storage manager‎‏‎‎‏‎"</string>
-    <string name="empty_state_title" msgid="4033285438176545309">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‏‎‏‏‏‏‏‏‏‏‎‎‏‎‎‎‏‏‎‏‏‏‎‏‏‏‏‎‎‏‎‎‏‏‏‎‎‎‏‎‎‎‏‎‎‏‏‎‎‏‎‏‎‎‎‎‏‏‏‎‏‎Nothing to remove‎‏‎‎‏‎"</string>
-    <string name="empty_state_review_items_link" msgid="8411186441239304545">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‏‎‎‏‎‏‏‏‎‏‎‏‎‎‎‏‎‎‏‏‎‎‎‎‎‎‏‎‎‎‎‎‎‎‏‏‎‏‎‎‏‏‎‏‎‎‏‏‎‎‏‎‏‏‎‎‎‎‏‎Review recent items‎‏‎‎‏‎"</string>
-    <string name="empty_state_summary" msgid="8439893007424243790">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‏‎‏‎‎‏‎‎‎‎‎‏‎‎‎‎‏‎‏‏‏‏‏‏‎‎‏‎‏‎‎‏‎‎‏‏‏‎‏‎‎‏‏‎‏‎‎‏‎‎‎‎‏‎‎‏‏‏‎‎There aren\'t any older files to remove. To make space, remove recent photos, videos, and apps.‎‏‎‎‏‎"</string>
-    <string name="app_requesting_space" msgid="857425181289960167">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‎‏‎‏‏‏‏‏‎‎‏‏‎‎‎‏‎‏‏‏‏‏‎‏‎‏‎‎‎‏‏‎‏‏‎‎‎‎‏‎‎‎‎‏‏‎‏‎‏‎‎‏‎‏‏‏‎‎‏‏‏‎‎‏‎‎‏‏‎<xliff:g id="APP">%1$s</xliff:g>‎‏‎‎‏‏‏‎ needs ‎‏‎‎‏‏‎<xliff:g id="CLEARABLE_BYTES">%2$s</xliff:g>‎‏‎‎‏‏‏‎ of space‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_automatic_title" msgid="4610119358490679299">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‏‏‏‏‏‏‏‏‏‏‎‏‎‎‏‏‎‏‏‏‏‎‎‏‎‎‎‎‏‏‎‎‎‏‏‏‎‏‏‏‏‏‏‎‏‎‎‎‏‏‎‎‎‎‎‎‎‎‎‏‏‎Automatic‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_manual_title" msgid="3905192395039688529">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‏‎‏‏‎‎‎‏‏‎‎‏‎‎‎‎‎‎‏‏‏‏‏‎‎‏‎‏‎‏‏‏‎‏‏‎‏‏‏‏‎‎‏‎‏‏‏‎‎‎‏‏‏‎‏‎‏‎‎‎‏‎Manual‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_preference_title" msgid="803484286872745892">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‎‏‎‏‏‎‎‏‎‎‏‏‎‏‎‎‎‏‏‎‎‏‎‏‏‎‎‏‏‏‎‏‎‏‏‎‏‏‎‏‎‏‎‎‎‎‏‏‏‎‏‏‏‏‎‏‎‎‏‎‎‎Free space now‎‏‎‎‏‎"</string>
+    <string name="automatic_storage_manager_service_label" msgid="648592272375278978">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‎‏‎‎‏‎‎‎‎‎‎‎‎‎‏‎‎‎‎‏‏‎‎‏‏‏‏‎‎‎‎‏‏‏‏‏‎‏‏‏‏‏‏‎‏‏‏‏‎‏‏‎‏‏‎‎‎‎‎‏‎‎Automatic Storage Management Service‎‏‎‎‏‎"</string>
+    <string name="automatic_storage_manager_notification_title" product="default" msgid="8480483265300964701">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‏‎‏‏‎‏‏‎‎‎‎‏‎‏‏‏‎‏‎‏‎‎‏‏‎‎‏‎‎‏‎‎‏‏‏‏‏‏‎‏‎‎‎‏‎‏‎‎‏‎‏‎‏‎‏‏‏‎‏‎Your phone is low on space‎‏‎‎‏‎"</string>
+    <string name="automatic_storage_manager_notification_title" product="tablet" msgid="3150907918531844467">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‎‏‎‏‏‏‎‏‏‏‎‏‎‎‏‎‎‎‏‏‎‎‎‎‎‏‏‏‎‏‏‎‎‎‏‎‎‎‎‎‎‏‏‏‎‎‏‎‏‏‎‎‏‎‏‏‏‎‎‏‏‎Your tablet is low on space‎‏‎‎‏‎"</string>
+    <string name="automatic_storage_manager_notification_summary" msgid="2267399886362245822">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‏‏‏‏‏‏‎‏‏‏‎‏‏‏‎‏‏‎‏‏‎‎‎‎‏‎‏‏‏‎‏‏‏‏‎‏‏‎‎‏‏‎‎‎‎‏‏‏‏‏‏‎‏‎‏‎‏‏‏‏‏‎‎Let Storage Manager free up space by automatically removing old photos and videos when your phone starts to get full again.‎‏‎‎‏‎"</string>
+    <string name="automatic_storage_manager_cancel_button" msgid="1826365037494962803">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‏‏‏‎‎‏‎‏‎‏‏‎‎‎‏‎‎‎‏‏‎‏‎‏‎‏‎‎‏‏‎‎‏‏‏‎‏‎‎‎‎‏‎‏‎‎‏‎‎‎‏‎‏‎‎‏‏‏‎‎‏‏‎No thanks‎‏‎‎‏‎"</string>
+    <string name="automatic_storage_manager_activate_button" msgid="6257070263061898041">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‎‏‎‏‏‎‏‏‎‏‎‏‎‏‏‎‎‏‎‏‎‎‎‏‎‎‎‎‎‎‏‎‏‏‏‏‎‎‎‏‏‏‎‏‏‎‏‏‏‎‎‎‏‏‎‎‏‏‏‎‎‏‎Turn on‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_photos_loading_title" msgid="2768067991066779772">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‎‎‏‏‎‎‏‏‎‏‎‏‎‎‎‏‎‎‏‏‏‎‎‏‎‏‏‎‎‎‎‎‎‏‎‎‏‏‎‏‎‏‎‏‎‎‎‏‎‎‎‎‎‎‏‏‏‏‏‎‎‎Backed up photos &amp; videos‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_photos_loading_summary" msgid="8203033249458245854">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‎‎‏‏‏‎‏‎‏‏‏‎‎‎‎‎‏‏‏‎‏‎‎‎‏‏‏‎‏‏‎‎‎‏‎‏‎‏‎‏‏‎‏‎‏‎‏‎‎‎‎‏‏‎‏‏‏‏‎‎Finding items…‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_no_threshold" msgid="6943179204098250444">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‎‎‎‎‎‏‎‏‏‎‏‏‎‎‏‎‎‎‎‎‏‎‏‏‎‎‏‏‎‎‎‏‏‎‎‎‎‎‏‎‎‏‏‏‏‏‎‏‎‏‏‎‏‏‎‎‏‏‎‎‎Show all items‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_default_threshold" msgid="8410389370069021113">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‏‎‎‏‎‏‏‎‏‏‏‏‎‏‏‎‏‎‎‏‎‎‏‎‎‏‎‎‏‏‎‏‎‎‏‏‎‏‎‏‏‏‏‎‏‏‏‎‎‎‏‏‎‏‏‏‎‎‏‎Hide recent items‎‏‎‎‏‎"</string>
+    <string name="deletion_helper_clear_dialog_message_first_time" msgid="686530413183529901">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‎‏‎‎‏‏‎‎‎‎‏‏‏‎‎‎‎‏‎‏‏‏‏‎‎‎‏‎‏‏‏‏‏‏‏‎‏‎‏‎‎‏‏‏‎‎‏‏‏‎‎‏‏‏‎‏‎‏‏‎‏‎‎‏‎‎‏‏‎<xliff:g id="CLEARABLE_BYTES">%1$s</xliff:g>‎‏‎‎‏‏‏‎ of content will be removed from your device‎‏‎‎‏‎"</string>
+    <string name="automatic_storage_manager_activation_warning" msgid="7657017408180001078">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‏‎‏‎‎‏‎‎‎‎‏‏‎‎‏‏‎‎‎‎‏‏‎‎‎‎‏‏‏‎‎‏‏‏‎‏‏‎‎‎‏‏‎‏‎‎‎‏‏‎‎‏‎‎‏‏‎‏‏‎‎Your storage is now being managed by the storage manager‎‏‎‎‏‎"</string>
+    <string name="empty_state_title" msgid="4033285438176545309">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‎‏‏‎‏‏‏‏‏‏‏‏‎‎‏‎‎‎‏‏‎‏‏‏‎‏‏‏‏‎‎‏‎‎‏‏‏‎‎‎‏‎‎‎‏‎‎‏‏‎‎‏‎‏‎‎‎‎‏‏‏‎‏‎Nothing to remove‎‏‎‎‏‎"</string>
+    <string name="empty_state_review_items_link" msgid="8411186441239304545">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‏‎‎‏‎‏‏‏‎‏‎‏‎‎‎‏‎‎‏‏‎‎‎‎‎‎‏‎‎‎‎‎‎‎‏‏‎‏‎‎‏‏‎‏‎‎‏‏‎‎‏‎‏‏‎‎‎‎‏‎Review recent items‎‏‎‎‏‎"</string>
+    <string name="empty_state_summary" msgid="8439893007424243790">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‏‎‏‎‏‎‎‏‎‎‎‎‎‏‎‎‎‎‏‎‏‏‏‏‏‏‎‎‏‎‏‎‎‏‎‎‏‏‏‎‏‎‎‏‏‎‏‎‎‏‎‎‎‎‏‎‎‏‏‏‎‎There aren\'t any older files to remove. To make space, remove recent photos, videos, and apps.‎‏‎‎‏‎"</string>
+    <string name="app_requesting_space" msgid="857425181289960167">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‎‎‎‎‏‎‏‏‏‏‏‏‏‎‎‏‎‏‏‏‏‏‎‎‏‏‎‎‎‏‎‏‏‏‏‏‎‏‎‏‎‎‎‏‏‎‏‏‎‎‎‎‏‎‎‎‎‏‏‎‏‎‏‎‎‏‎‏‏‏‎‎‏‏‏‎‎‏‎‎‏‏‎<xliff:g id="APP">%1$s</xliff:g>‎‏‎‎‏‏‏‎ needs ‎‏‎‎‏‏‎<xliff:g id="CLEARABLE_BYTES">%2$s</xliff:g>‎‏‎‎‏‏‏‎ of space‎‏‎‎‏‎"</string>
 </resources>
diff --git a/res/values-es/strings.xml b/res/values-es/strings.xml
index 2fda576..e1e00d0 100644
--- a/res/values-es/strings.xml
+++ b/res/values-es/strings.xml
@@ -62,6 +62,6 @@
     <string name="automatic_storage_manager_activation_warning" msgid="7657017408180001078">"El Administrador de Almacenamiento gestiona ahora el almacenamiento"</string>
     <string name="empty_state_title" msgid="4033285438176545309">"No hay nada que eliminar"</string>
     <string name="empty_state_review_items_link" msgid="8411186441239304545">"Revisar elementos recientes"</string>
-    <string name="empty_state_summary" msgid="8439893007424243790">"No hay archivos más antiguos que se puedan eliminar. Para liberar espacio, borra aplicaciones, fotos y vídeos recientes."</string>
+    <string name="empty_state_summary" msgid="8439893007424243790">"No hay archivos antiguos que se puedan eliminar. Para liberar espacio, borra aplicaciones, fotos y vídeos recientes."</string>
     <string name="app_requesting_space" msgid="857425181289960167">"<xliff:g id="APP">%1$s</xliff:g> necesita <xliff:g id="CLEARABLE_BYTES">%2$s</xliff:g> de espacio"</string>
 </resources>
diff --git a/res/values-nl/strings.xml b/res/values-nl/strings.xml
index 6ce445a..8cd9064 100644
--- a/res/values-nl/strings.xml
+++ b/res/values-nl/strings.xml
@@ -26,7 +26,7 @@
     <string name="deletion_helper_app_summary_never_used" msgid="2695210890063792461">"Niet gebruikt in het afgelopen jaar"</string>
     <string name="deletion_helper_app_summary_unknown_used" msgid="8576377054665785558">"Laatste gebruiksdatum onbekend"</string>
     <string name="deletion_helper_free_button" msgid="1760529213407548661">"<xliff:g id="FREEABLE">%1$s</xliff:g> vrijmaken"</string>
-    <string name="deletion_helper_photos_title" msgid="2602723121486729972">"Back-up van foto\'s en video\'s"</string>
+    <string name="deletion_helper_photos_title" msgid="2602723121486729972">"Opgeslagen foto\'s en video\'s"</string>
     <string name="deletion_helper_photos_age_summary" msgid="1820871709448371984">"Meer dan 30 dagen oud"</string>
     <!-- no translation found for deletion_helper_photos_summary (3194825464552700889) -->
     <skip />
diff --git a/robotests/src/com/android/storagemanager/automatic/AutomaticStorageManagementJobServiceTest.java b/robotests/src/com/android/storagemanager/automatic/AutomaticStorageManagementJobServiceTest.java
index 736d7b4..27bd7c8 100644
--- a/robotests/src/com/android/storagemanager/automatic/AutomaticStorageManagementJobServiceTest.java
+++ b/robotests/src/com/android/storagemanager/automatic/AutomaticStorageManagementJobServiceTest.java
@@ -191,8 +191,8 @@ public class AutomaticStorageManagementJobServiceTest {
     }
 
     @Test
-    public void testASMJobRunsWithValidConditionsIfOptInNotShownButUnactivated() {
-        StorageManagerShadowSystemProperties.put("ro.storage_manager.show_opt_in", "false");
+    public void testASMJobRunsWithValidConditionsIfEnabledByDefaultAndUnset() {
+        StorageManagerShadowSystemProperties.put("ro.storage_manager.enabled", "true");
         assertThat(mJobService.onStartJob(mJobParameters)).isFalse();
         assertStorageManagerJobRan();
     }
diff --git a/src/com/android/storagemanager/automatic/NotificationController.java b/src/com/android/storagemanager/automatic/NotificationController.java
index 58c99d8..18e4533 100644
--- a/src/com/android/storagemanager/automatic/NotificationController.java
+++ b/src/com/android/storagemanager/automatic/NotificationController.java
@@ -98,12 +98,14 @@ public class NotificationController extends BroadcastReceiver {
     public void onReceive(Context context, Intent intent) {
         switch (intent.getAction()) {
             case INTENT_ACTION_ACTIVATE_ASM:
-                Settings.Secure.putInt(context.getContentResolver(),
+                Settings.Secure.putInt(
+                        context.getContentResolver(),
                         Settings.Secure.AUTOMATIC_STORAGE_MANAGER_ENABLED,
                         1);
                 // Provide a warning if storage manager is not defaulted on.
                 if (!SystemProperties.getBoolean(STORAGE_MANAGER_PROPERTY, false)) {
                     Intent warningIntent = new Intent(context, WarningDialogActivity.class);
+                    warningIntent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
                     context.startActivity(warningIntent);
                 }
                 break;
-- 
2.17.1

